portal-kernel/src/com/liferay/exportimport/kernel/lar/StagedModelDataHandler.java

```diff
@@ -28,29 +28,114 @@
  */
 public interface StagedModelDataHandler<T extends StagedModel> {
 
+	/**
+	 * Deletes a staged model based on the parameters. It is called when
+	 * deletions are imported.
+	 *
+	 * @param  uuid the uuid of the staged model
+	 * @param  groupId the group ID of the entity's group
+	 * @param  className the name of the staged model class
+	 * @param  extraData can be any useful information about the staged model.
+	 *         This information makes the staged model easier to identify /
+	 *         fetch for deletion. It is populated when a deletion system event
+	 *         is added usually in the <code>*LocalServiceImpl</code> class of
+	 *         the model.
+	 * @throws PortalException
+	 */
 	public void deleteStagedModel(
 			String uuid, long groupId, String className, String extraData)
 		throws PortalException;
 
+	/**
+	 * Deletes the given staged model
+	 *
+	 * @param  stagedModel staged model to be deleted
+	 * @throws PortalException
+	 */
 	public void deleteStagedModel(T stagedModel) throws PortalException;
 
+	/**
+	 * Exports a given staged model and its references. See {@link
+	 * BaseStagedModelDataHandler#exportStagedModel(PortletDataContext, T)} for
+	 * reference implementation. Normally this method should not be overridden,
+	 * but {@link
+	 * BaseStagedModelDataHandler#doExportStagedModel(PortletDataContext, T)}.
```

Comment:
@matethurzo I've restructured the last sentence to say the following:

> Refrain from overriding this method, and instead, override the `BaseStagedModelDataHandler.doExportStagedModel` method.

Is that correct?